dunwoody_disaster/Item.py

```diff
@@ -67,80 +69,77 @@ def widget(self, min_width=100) -> QWidget:
         return widget
 
     def to_dict(self) -> dict:
-        DD.unimplemented()
-        return {}
+        return {
+            "name": self.name,
+            "damage": self.damage,
+            "magic": self.magicCost,
+            "stamina": self.staminaCost,
+        }
 
 
 class Weapon(Item):
-    def __init__(self, name, magic, damage, magicCost, staminaCost):
-        super().__init__(name)
-        try:
-            self.stats = WeaponStats[name]
-        except KeyError:
-            self.stats = [0, 0, 0]
-        self.magic = magic
-        self.damage = damage
-        self.magicReq = magicCost
-        self.staminaCost = staminaCost
+    def __init__(self, name: str, magical: bool, damage: int, stamina: int, magic: int):
+        super().__init__(name, damage, stamina, magic)
+        self.magical = magical
+
+    def to_dict(self) -> dict:
+        result = super().to_dict()
+        result["magical"] = self.magical
+        return result
 
     @staticmethod
     def from_json(json: dict) -> "Weapon":
         return Weapon(
             json["name"],
-            json["is_magical"],
+            json["magical"],
             json["damage"],
-            json["magicCost"],
-            json["staminaCost"],
+            json["stamina"],
+            json["magic"],
         )
 
     @staticmethod
     def default() -> "Weapon":
-        weapon = Weapon("Fist", 0, 1, 0, 0)
+        weapon = Weapon("Fist", False, 1, 0, 0)
         return weapon
 
-    def to_dict(self) -> dict:
-        return {
-            "name": self.name,
-            "is_magical": self.magic,
-            "damage": self.damage,
-            "magicCost": self.magicReq,
-            "staminaCost": self.staminaCost,
-        }
-
 
-class Food(Item):
-    def __init__(self, name):
-        super().__init__(name)
-        self.stats = FoodStats[name]
+# class Food(Item):
+#     def __init__(self, name):
+#         super().__init__(name)
```

Comment:
We should make a Trello card for this to be implemented...

Reply:
@MLHumphries, think you could add it under card [#81](https://trello.com/c/Rbgv2k0K)?